src/hooks/useOpenOrdersFetch.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { userOpenOrdersFetch } from '../modules/user/openOrders';
+import { selectUserLoggedIn } from '../modules/user/profile';
+
+export const useOpenOrdersFetch = ({ id }) => {
```

Comment:
Pass whole market object here
```suggestion
export const useOpenOrdersFetch = ({ market: Market }) => {
```